src/styles/index.scss

```diff
@@ -1,3 +1,88 @@
+$stopwatcDmtr: 80vmin;
```

Comment:
There is a typo in the variable name `$stopwatcDmtr`. It should likely be `$stopwatchDmtr` to reflect the intended meaning.